pkg/sm/sm.go

```diff
@@ -120,6 +122,8 @@ func New(ctx context.Context, env env.Environment) *ServiceManagerBuilder {
 // Build builds the Service Manager
 func (smb *ServiceManagerBuilder) Build() *ServiceManager {
 	// setup server and add relevant global middleware
+	smb.installHealth()
```

Comment:
this is a slight issue, because the sbproxy doesn't use ServiceManagerBuilder, but SBProxyBuilder, so it can't get the default behaviour. Basically this same code should be copied in SBProxyBuilder...

Reply:
Thats correct - we discussed it also in terms of the race conditions fix - its not ideal - probably in a future pull request it wouls make sense to extract a generic application that can be built and contains all the duplicated stuff